### PR TITLE
priority: use SF terms, not ABNF

### LINKED ISSUE #1975: Use SF terms, not ABNF
Priority currently uses sf- prefixed ABNF to describe datatypes in prose. As discussed, this should use the terminology from the draft instead (e.g., "Integer, Boolean").

### PR DESCRIPTION
**Pull request by LPardue**
Closes #1975.

We're not in AUTH48 yet. I suggest we hold merging the change until that time.

### CODE REVIEW

File: draft-ietf-httpbis-priority.md [hunk 1]
```diff
@@ -130,8 +130,8 @@ consideration and guidance about how servers might act upon signals.
 
 {::boilerplate bcp14-tagged}
 
-The terms Dictionary, sf-boolean, sf-dictionary, and sf-integer are imported from
-{{!STRUCTURED-FIELDS=RFC8941}}.
+The terms Boolean, Dictionary, and Integer, along with the sf-dictionary rule,
```
Comment (martinthomson) [inline draft-ietf-httpbis-priority.md:133]: I don't think that you want to say that RFC 8941 defines the term "Boolean".  Maybe instead "This document uses the Boolean, Dictionary, and Integer types from {{!SF}} along with the `sf-dictionary` ABNF rule."

Comment (LPardue) [reply draft-ietf-httpbis-priority.md:133]: Thanks. "Boolean" is so generic, it was part of the reason why I gravitated to `sf-boolean` in prose in the first place.

Comment (mnot) [reply draft-ietf-httpbis-priority.md]: In prose, you can try "Boolean Structured Type" or similar.

Comment (LPardue) [reply draft-ietf-httpbis-priority.md]: The pattern I've seen so far is people saying "Structured Fields Dictionary" or "Structured Fields List". So "Structured Fields Boolean" would be consistent with that but its a bit wordy. Perhaps settling on a common acronym of `SF` in front of these would work.

Comment (LPardue) [reply draft-ietf-httpbis-priority.md]: Applying this to "SF Member" would also be help in specs that need to talk generically about the things in List or Dictionary without wanting to have to make up a term of their own.

File: draft-ietf-httpbis-priority.md [hunk 2]
```diff
@@ -328,7 +329,8 @@ responses that have impact on user interaction.
 
 ## Incremental
 
-The incremental parameter (`i`) takes an sf-boolean as the value that indicates
+The incremental parameter (`i`) takes a Boolean (see
+{{Section 3.3.6 of STRUCTURED-FIELDS}}) as the value that indicates
```
Comment (ioggstream) [inline draft-ietf-httpbis-priority.md:333]:
```suggestion
The incremental parameter (`i`) is a Boolean (see
{{Section 3.3.6 of STRUCTURED-FIELDS}}) that indicates
```
